dependencies/buster/redfish_client/control

```diff
@@ -0,0 +1,27 @@
+Source: ruby-redfish-client
+Section: ruby
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: Lukas Zapletal <lzap+pub@redhat.com>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               git,
+               ruby-excon (>= 0.60),
+               ruby-server-sent-events (>= 0.1)
```

Comment:
I don't think those are build dependencies?

Reply:
Hmm weird `gem2deb` added these. Removing.

Reply:
Hah. I spoke to quickly. Sorry. We run tests on debian, and that needs those, of course!